### ISSUE TITLE
EJS language syntax is not being highlighted

### ISSUE THREAD

**Issue by JoemerRamos** [Digitalbrainstem/ejs-grammar#81]
## Steps to reproduce:
Here's the line of code that isn't working and the settings I have set
![image](https://user-images.githubusercontent.com/34356209/119149538-9dd81700-ba1b-11eb-9183-ff2600b8a969.png)
![image](https://user-images.githubusercontent.com/34356209/119149608-adeff680-ba1b-11eb-81b9-c34887bac20b.png)

## Expected result:
Syntax of <% and %> should be highlighted

**Comment by fox11q** [comment]
Hello, I got this problem but still can't find right decision
![image](https://user-images.githubusercontent.com/90627229/140625851-02fff45e-6d9e-47f0-a613-35a0fd20324d.png)
Also my settings
![image](https://user-images.githubusercontent.com/90627229/140625869-af9ecb76-f9ed-4978-80d7-b4ec5805077d.png)
may be you can give me some advice how I can to fix that.

**Comment by fox11q** [comment]
I fixed it by installing EJS language supporter and it works but the syntax still don't highlighted

**Comment by Betanu701** [comment]
What is your language for the file set to? It should be HTML. 

The brackets do not highlight but the code within should be seen as javascript.

**Comment by Betanu701** [comment]
Here is what I see when I wrote a quick test with your code.

![image](https://user-images.githubusercontent.com/5798012/140626220-191eb3ca-1ee0-465a-9349-b8b4f61a9809.png)

**Comment by fox11q** [comment]
> What is your language for the file set to? It should be HTML.
> 
> The brackets do not highlight but the code within should be seen as javascript.

![image](https://user-images.githubusercontent.com/90627229/140626212-a22688ba-2794-4ae8-8675-edce8c053f41.png)
Is it correct highlights? I thought that the brackets also should be colorised

**Comment by fox11q** [comment]
> Here is what I see when I wrote a quick test with your code.
> 
> ![image](https://user-images.githubusercontent.com/5798012/140626220-191eb3ca-1ee0-465a-9349-b8b4f61a9809.png)

I understood. Thx you man)

**Comment by Betanu701** [comment]
Yup, everything looks to be correct to me. Glad it is fixed for you!
